components/resourceDetails/resourceSharing/agentAccess/agentAccessOptionsMenu/removeButton/index.jsx

```diff
@@ -80,9 +81,13 @@ export const handleRemovePermissions = ({
     if (AUTHENTICATED_AGENT_PREDICATE === agentWebId) {
       accessControl.setRuleAuthenticated(policyName, false);
     }
-    await accessControl.removeAgentFromNamedPolicy(agentWebId, policyName);
+    const { response: updatedAcr } = await accessControl.removeAgentFromPolicy(
+      agentWebId,
+      policyName
+    );
+    await mutateResourceInfo(updatedAcr, false);
     setLoading(false);
-    await mutatePermissions();
+    // await mutatePermissions();
```

Comment:
Remove?

Reply:
Yes, not needed with the current solution